<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="redEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="greenEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="blueEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>411</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="colorLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateRed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateGreen(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateBlue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateRedEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateGreenEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateBlueEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateEditBox(int)</slot>
  <slot>updateRed(int)</slot>
  <slot>updateGreen(int)</slot>
  <slot>updateBlue(int)</slot>
  <slot>updateRedEdit()</slot>
  <slot>updateGreenEdit()</slot>
  <slot>updateBlueEdit()</slot>
 </slots>
</ui>
